rllib/algorithms/algorithm_config.py

```diff
@@ -2256,6 +2256,57 @@ def is_policy_to_train(pid, batch=None):
 
         return policies, is_policy_to_train
 
+    def validate_train_batch_size_vs_rollout_fragment_length(self) -> None:
```

Comment:
stupid questions: 1) why do we need to be able to specify both that roughly match each other? why not just error out when train_batch_size does not match the expected value based on the rollout_fragment_length value? 2) is setting rollout_fragment_length = "auto" always recommended?

Reply:
Well, for off-policy algos, `rollout_fragment_length` can be whatever it wants and it is not linked to the train batch size. For on-policy, I'm thinking that sometimes, users would want to set the rollout fragment length manually to force a certain rollout behavior, however, through this new error, we force them to be aware that this will have an effect on their train batch size.